PatchesAndAreasApi/V1/Infrastructure/CompareValues.cs

```diff
@@ -0,0 +1,44 @@
+using System.Collections.Generic;
+using System;
+using Hackney.Shared.PatchesAndAreas.Domain;
+
+namespace PatchesAndAreasApi.V1.Infrastructure
+{
+    public static class CompareValues
```

Comment:
Is this used anywhere? Looks like you used the .AreAllSame method instead

Reply:
Oh, I didn't know you could use a method call for a different class like that in C#, very cool! Naming is fine.